sphere-common/src/main/java/io/sphere/sdk/queries/IsGreaterThanOrEqualsQueryPredicate.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.queries;
+
+class IsGreaterThanOrEqualsQueryPredicate<T, V, M> extends ComparisonQueryPredicate<T, V, M> {
+
+    public IsGreaterThanOrEqualsQueryPredicate(final QueryModel<M> queryModel, final V value) {
```

Comment:
Now that we are renaming these predicates, a small discussion regarding the naming:
"price is greater than 3" or "price equals 4" I think are both correct (not sure how natural the second one is in this sentence, though).
But I think you would always say "price is greater than or equal to 4". At least that's the reason why I wrote it like this in the Search API: https://github.com/sphereio/sphere-jvm-sdk/blob/master/sphere-common/src/main/java/io/sphere/sdk/search/RangedFacetSearchModel.java#L54

So from my humble knowledge of English grammar I would use the "equal to" version because that's what I found people use when talking. What version is more normal to you?